server/controllers/stock/reports/purchaseOrderAnalysis/report.handlebars

```diff
@@ -74,65 +74,61 @@
         {{#each this.inventoriesOrdered}}
           <br>
           <br>
-          <table style="width: 90%; margin-right: auto; margin-left: auto" class="table table-striped table-condensed table-report table-bordered">
-
-              <tr>
-                <td width="15%" class="text-uppercase"> {{translate 'FORM.LABELS.CODE'}}  </td>
-                <td colspan="7" width="85%"><strong> {{ code }} </strong></td>
-              </tr>
-              <tr>
-                <td width="15%" class="text-uppercase"> {{translate 'FORM.LABELS.INVENTORY'}}  </td>
-                <td colspan="7" width="85%"><strong> {{ inventory_text }} </strong></td>
+          <table style="width: 90%; margin: auto;" class="table table-condensed table-report table-bordered">
```

Comment:
I do not suggest this be fixed now, but in general the html for tables should use <caption><thead>, <th>, <tbody>, etc.  See https://alistapart.com/article/web-typography-tables/.    Using these extra elements allows more uniform and appropirate application of CSS rules.    If that is done, rather than embedding 'text-bold' in the <tr> for the header, simple CSS rules for <th> can used and changed later without affecting the HTML code.